src/App/App.js

```diff
@@ -1,12 +1,28 @@
 import './App.css';
-import { Routes, Route } from "react-router-dom"
+import { Routes, Route } from "react-router-dom";
+import { useState, useEffect } from 'react';
 import Home from "../Home/Home";
 import Details from "../Details/Details";
-import mockData from '../mockData'
+import Nav from "../Nav/Nav";
+import Form from "../Form/Form";
+import Card from "../Card/Card";
+import mockData from '../mockData';
 
 const App = () => {
 
-  //need a fetch request to get all the parks on page load
+  const [allParks, setAllParks] = useState(null)
+
+  useEffect(() => {
+    fetch(`https://developer.nps.gov/api/v1/parks?parkCode=&limit=471&start=0&api_key=l6jn2TRgOT3bXFR8Fk7iAF7OP6Bkf7lslJE9TMMX`)
+      .then((res) => res.json())
+      .then((res) => {
+        setAllParks(res)
+      })
```

Comment:
I'd probably set up a much smaller set of mock data to use locally so that you don't run into those rate limits - that's a LOT of parks you're loading with each change